assets/armsnippets.jsonc

```diff
@@ -1818,14 +1820,16 @@
             "{",
             "    \"name\": \"${1:storageaccount1}\",",
             "    \"type\": \"Microsoft.Storage/storageAccounts\",",
-            "    \"apiVersion\": \"2015-06-15\",",
-            "    \"location\": \"[resourceGroup().location]\",",
+            "    \"apiVersion\": \"2019-06-01\",",

```

Comment:
I'd stick to the property ordering suggested in https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#best-practices, except perhaps for name, which we've consistently got first?  Maybe we should change those eventually?

Reply:
I will create a work item to scrub / update the complete tempalte file to align with the best practices doc.

Reply:
https://github.com/microsoft/vscode-azurearmtools/issues/525